src/llama-graph.h

```diff
@@ -242,7 +243,7 @@ class llm_graph_input_attn_kv_unified : public llm_graph_input_i {
         cparams(cparams),
         kv_state(kv_state) {
     }
-    ~llm_graph_input_attn_kv_unified() = default;
+    virtual ~llm_graph_input_attn_kv_unified() = default;
```

Comment:
I don't think this `virtual` is needed. I think I added it when I was attempting to have the hybrid input inherit from the unified input